.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.eslintignore
```

Comment:
I think it would be beneficial for all of us to have .eslintignore file version controlled in the project. Please remove this line.

Reply:
Yeah, I agree we have the same ignore so all we on same standard and code format as well